contribute.md

```diff
@@ -8,50 +8,50 @@ This page is dedicated as a reference for RE:SS3D contributors.
 
-1. Go to the appropriate board/list for your field and look for available tasks
-2. Pick a task, and request a @Centcom member to assign you to that task
-3. Work on that task, using the correct style and formatting
-4. Post in-progress screenshots or videos to #wip for extra praise
-5. Submit the task (as a PR for code, or by posting the asset to discord and notifying someone in @Centcom)
-6. They will review it, and may request changes
-7. Submit screenshots and/or videos to the #devblog channel along with a link for an asset or a summary for code
-8. Once accepted, pat yourself on the back for your glorious contribution to SS3D
+1. Go to the appropriate board/list for your field and look for available tasks.
+2. Pick a task, and request a @Centcom member to assign you to that task.
+3. Work on that task, using the correct style and formatting.
+4. Post in-progress screenshots or videos to #wip for extra praise.
+5. Submit the task (as a PR for code, or by posting the asset to discord and notifying someone in @Centcom).
+6. They will review it, and may request changes.
+7. Submit screenshots and/or videos to the #devblog channel along with a link for an asset or a summary for code.
+8. Once accepted, pat yourself on the back for your glorious contribution to SS3D.
 
 ### How to contribute
 
-If you're a programmer, an artist, a soundcloud rapper, a programmer, none of the above, or *especially* a programmer, we need you!
-For coding and modelling tasks we have a [task board](https://github.com/RE-SS3D/SS3D/projects/2) (the ones in the available column are ones you can start on) and a [model list](https://trello.com/b/ZVcDitv0/ss3d-model-list).
+If you're a programmer, 2D artist, 3D modeler, musician, programmer, animator, soundcloud rapper, or *especially* a programmer, we need you!
+For coding tasks we have a [task board](https://github.com/RE-SS3D/SS3D/projects/2) (the ones in the available column are ones you can start on). For more info on pull requests or reporting bugs check out the [contributing](https://github.com/RE-SS3D/SS3D/blob/master/CONTRIBUTING.md) page on our github.
+and a [model list](https://trello.com/b/ZVcDitv0/ss3d-model-list).
 To start on either just go to our discord and request an admin or moderator to assign you to a task you can see on there.
 **But before you do please check the style guide below**.
 
-For any other kind of task, e.g. sounds, concept art, or emotional support (please help me) just ask around on the discord and someone
-can help you out.
+For any other kind of task, e.g. sounds, concept art, or emotional support (please help me) just ask around on the discord and someone can help you out.
 
 ### Style guide & Kits
 
-**The code style guide** can be found on our [github](https://github.com/RE-SS3D/SS3D/blob/develop/StyleGuides/C_SHARP.md). It contains:
+**The code style guide** can be found on our [here](https://github.com/RE-SS3D/SS3D/blob/master/StyleGuides/C_SHARP.md). It contains:
```

Comment:
here should probably be github. Currently reads: "can be found on our here"

Reply:
I used "here" because every other instance of "github" as a link goes to our github home page. I did fix the grammar issue as well as some others I found.